.github/workflows/tests.yaml

```diff
@@ -363,6 +363,40 @@ jobs:
         source: /tmp/artifacts
       if: always()
 
+  scuba-tests:
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      S3BACKEND: mem
+      SCUBA_HOST: localhost
+      SCUBA_PORT: 8100
+      SCUBA_HEALTHCHECK_FREQUENCY: 100
+      CLOUDSERVER_IMAGE: ghcr.io/${{ github.repository }}/cloudserver:${{ github.sha }}
+      MONGODB_IMAGE: ghcr.io/${{ github.repository }}/ci-mongodb:${{ github.sha }}
+      JOB_NAME: ${{ github.job }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Setup CI environment
+      uses: ./.github/actions/setup-ci
+    - name: Setup CI services
+      run: docker compose up -d
+      working-directory: .github/docker
+    - name: Run scuba tests
+      run: |-
+        set -ex -o pipefail;
+        bash wait_for_local_port.bash 8000 40
+        yarn run test_scuba | tee /tmp/artifacts/${{ github.job }}/tests.log
```

Comment:
even with the function tests, we actually don't test with the "real" scuba : so we may easily introduce issue if the mock does not match the actual scuba... → we should find a way to run these tests with either/both mock scuba or "real" scuba

Reply:
The functional tests are only testing cloudserver contract here, but yes, to me this is a second step (as we'll have the integration test in Zenko), in the same way we test Vault + MongoDB today, we can really test with scuba, what do you think?

Reply:
Created https://scality.atlassian.net/browse/CLDSRV-528, in the improvement epic. It will be about not only scuba, but also vault (we don't test it today in the CI...)